installation_scripts/install_Ubuntu.sh

```diff
@@ -8,11 +8,51 @@
 # You use it at your own risk. 
 # We assume no liability for the accuracy, correctness, completeness, or usefulness of this script, nor for any sort of damages that using it may cause.
 
+# --------
+# SETTINGS
+# --------
+
+hbdir=~/.linuxbrew
+bashfile=~/.bashrc
+
+echo ""
+if [ $# -eq 0 ]; then
+  echo "Using default installation and file paths."
+fi
+
+# http://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+while [[ $# > 1 ]]; do
+  key="$1"
+
+  case $key in
+    -p|--prefix)
+      hbdir="$2"
+      shift # past argument
+    ;;
+    -b|--bashfile)
+      bashfile="$2"
+      shift # past argument
+    ;;
+    --default)
```

Comment:
If you don't feel strongly about it, i would remove `--default` option since you don't do anything there. It's up to you.